src/pages/Profile/EditMoto.js

```diff
@@ -0,0 +1,50 @@
+import axios from '../../../../axios';
+import { useHistory } from 'react-router-dom';
+import MotorcycleForm from '../MotorcycleForm';
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import useAuth from '../../../../hooks/useAuth';
+
+const EditMoto = props => {
+  const [auth] = useAuth();
+  const { id } = useParams();
+  const history = useHistory();
+  const [motorcycle, setMotorcycle] = useState(null);
+
+  const submit = async form => {
+    await axios.patch(`/motorcycles/${id}.json?auth=${auth.token}`, form);

```

Comment:
The logic with "auth=${auth.token}" should be placed in a more generic place. When you log in you should automatically add auth param to all requests.

Reply:
Ok, I will try to fix it